packages/aws-cdk-lib/aws-stepfunctions/lib/activity.ts

```diff
@@ -57,14 +80,38 @@ export class Activity extends Resource implements IActivity {
    */
   public readonly activityName: string;
 
+  /**
+   * @attribute
+   */
+  public readonly kmsKey: kms.IKey | undefined;
```

Comment:
Why the union type here and not just `  public readonly kmsKey: kms.IKey`?

Reply:
Will update to: `public readonly kmsKey?: kms.IKey;`